### PR TITLE
Fixed smooth shuttle walls not smoothing properly on rotation.

### PR DESCRIPTION
**Pull request by xxalpha**
Fixes https://github.com/tgstation/tgstation/issues/18097

### CODE REVIEW

File: code/game/turfs/simulated/walls_misc.dm [hunk 1]
```diff
@@ -158,8 +158,10 @@
 	T.transform = transform
 
 
-//why don't shuttle walls habe smoothwall? now i gotta do rotation the dirty way
+//why don't shuttle walls habe smoothwall? now i gotta do rotation the dirty way <- DOUBLE NIGGER FOR NOT CALLING PARENT
```
Comment (Iamgoofball) [inline code/game/turfs/simulated/walls_misc.dm]: can we not have slurs in the codebase

Comment (xxalpha) [inline code/game/turfs/simulated/walls_misc.dm]: But all you code is a slur.

**Comment by ChangelingRain** [comment]
This does not fix #12130, as it is not smoothing-related.

**Comment by Iamgoofball** [comment]
:+1: its acceptable now

**Comment by Bawhoppen** [comment]
reeeeeee

**Pull request merged**